roles/foreman_client_repositories/tasks/release_repo.yml

```diff
@@ -4,3 +4,6 @@
     name: https://yum.theforeman.org/client/{{ foreman_client_repositories_version }}/{{ foreman_client_repositories_dists[ansible_os_family] }}{{ ansible_distribution_major_version }}/x86_64/foreman-client-release.rpm
     disable_gpg_check: True
     state: present
+  when:
+    - katello_repositories_version == "nightly" or foreman_client_repositories_version is version('3.9', '<')
```

Comment:
`katello_repositories_version` is not necessarily defined in the client context.